src/privatesend-client.cpp

```diff
@@ -1182,25 +1182,38 @@ bool CPrivateSendClientSession::SubmitDenominate(CConnman& connman)
     std::vector<CTxOut> vecTxOutRet;
     // lean towards "highest" branch but still mix via "lowest" one someties
     bool fMixLowest = GetRandInt(4) == 0;
+    // lean towards edges but still mix starting from the middle someties
+    // Note: liqudity providers always start from 0
+    int nRoundStart = GetRandInt(4) == 0
+                        ? (privateSendClient.nLiquidityProvider ? 0 : (privateSendClient.nPrivateSendRounds / 2))
+                        : (fMixLowest ? 0 : privateSendClient.nPrivateSendRounds);
```

Comment:
Can you convert this into if/else statements? Hard to read and using if/else would make this easier.

Reply:
Not exactly if/else but should be simpler to read now imo.

Reply:
What about this?
```diff
diff --git a/src/privatesend-client.cpp b/src/privatesend-client.cpp
index bf1d60f2f..ac0f74fe1 100644
--- a/src/privatesend-client.cpp
+++ b/src/privatesend-client.cpp
@@ -1185,9 +1185,13 @@ bool CPrivateSendClientSession::SubmitDenominate(CConnman& connman)
     // lean towards edges but still mix starting from the middle someties
     // Note: liqudity providers always start from 0
     bool fScanFromTheMiddle = (privateSendClient.nLiquidityProvider == 0) && (GetRandInt(4) == 0);
-    int nRoundStart = fScanFromTheMiddle
-                        ? privateSendClient.nPrivateSendRounds / 2
-                        : (fMixLowest ? 0 : privateSendClient.nPrivateSendRounds);
+
+    int nRoundStart = privateSendClient.nPrivateSendRounds;
+    if (fScanFromTheMiddle) {
+        nRoundStart = privateSendClient.nPrivateSendRounds / 2;
+    } else if (fMixLowest) {
+        nRoundStart = 0;
+    }
 
     // Submit transaction to the pool if we get here
     if (privateSendClient.nLiquidityProvider || fMixLowest) {
```

I find this much more clear, as it's immediately visible that there are 3 possible cases. With the nested ternary operators, it's very hard to see this.

Reply:
Ok, makes sense. replaced with e0b0f4e